src/commands/dsyms/api.ts

```diff
@@ -0,0 +1,43 @@
+import {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios'
+import FormData from 'form-data'
+import fs from 'fs'
+import {Writable} from 'stream'
+
+import {buildPath, getRequestBuilder} from '../../helpers/utils'
+
+import {Payload} from './interfaces'
+import {renderUpload} from './renderer'
+import {zip} from './utils'
+
+const maxBodyLength = Infinity
+
+export const uploadDSYM = (request: (args: AxiosRequestConfig) => AxiosPromise<AxiosResponse>) => async (
+  dSYM: Payload,
+  write: Writable['write']
+) => {
+  const form = new FormData()
+  write(renderUpload(dSYM))
+  form.append('type', dSYM.type)
+  const concatUUIDs = dSYM.uuids.join()
+  form.append('uuids', concatUUIDs)
+
+  const tmpFilePath = buildPath('/tmp', 'datadog-ci', 'dsyms', `${concatUUIDs}.zip`)
```

Comment:
i switched to `tmpdir()` (results in `/var/{random string}` folders in mac OS) 👍 
but i kept appending timestamp to the folder name instead of using `mkdtemp`'s 6 random digits.

imo timestamp might make debugging slighlty easier. do you think i still should use `mkdtemp`?

Reply:
Nope, timestamps are fine in my opinion.